### DISCUSSION TITLE
Run repurposing use case for all rare disease identifiers

### DISCUSSION THREAD

**Discussion by southalln** [NCATSTranslator/Relay#123] [Ideas]
Bridging this conversation from https://github.com/NCATSTranslator/Relay/issues/82

Naive idea is to capture TRAPI questions that help surface potential repurposing opportunities, and then run them for a large set of diseases. How best to develop this idea?

**Comment by southalln** [comment]
For example, we could start with inborn errors of metabolism as a specific class, and ask [using TRAPI] whether the accumulation of substrate causes adverse health outcomes (develop a substrate reduction therapy) or the lack of a product causes adverse health outcomes and whether there are ways to supplement that (nutritional support, inhibition of catabolism, activation of backup pathways).

**Comment by andrewsu** [comment]
I think a broad-based set of repurposing examples would be great for regression testing _if_ we had a gold (or silver) standard.  As we refine our ranking/filtering/sorting methods, it would be great to have a quantitative readout on the effect.

I think the primary question is how to assemble that gold standard.  I see three options:

* Is anyone aware of a **database of rare disease / orphan drug indications**?  That would be easiest to use, but of course that resource would have to be excluded from any ARA/KP in the test.  
* We could **scrape/curate unstructured or semistructured resources**.  For example, both orphanet ([example](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=197&Disease_Disease_Search_diseaseGroup=Lesch-Nyhan-Syndrome&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Lesch-Nyhan-syndrome&title=Lesch-Nyhan%20syndrome&search=Disease_Search_Simple)) and NORD ([example](https://rarediseases.org/rare-diseases/lesch-nyhan-syndrome/#standard-therapies)) have sections of therapies.  Also, several years ago someone in my lab looked into getting indications from FDA and EMA resources ([repo](https://github.com/stuppie/ema)).
* We could figure out a way to assemble **one-off cases from the literature and/or our institutions**.  The unsecret agent put two great use cases at https://github.com/webyrd/mediKanren/tree/master/biolink/use-cases (and I suspect they may have more).  We stumbled across one described [here](https://github.com/biothings/biothings_explorer/issues/135). We could also go back to some of the examples used at the ISB hackathon (I remember our SKS / mTOR inhibitor example, but there were certainly many others).

**Reply by southalln** [reply]
We have the FDA orphan-designated products (the disease and development status), but not a great way to pull them out by mode of action (substrate-reduction, or inhibition of catabolism, etc.), but we do know what treatments have been declared to treat rare diseases. If we focus on inborn errors of metabolism, I can list famous, specific examples --- tyrosinemia, alkaptonuria, Urea Cycle, Niemann Pick, etc.

**Reply by southalln** [reply]
And those documented use-cases from (Un)Secret and BTE look to be very nice real-world examples of what the intent of translator is

**Reply by cbizon** [reply]
I absolutely think we should start focusing on a particular case as a way to drive things, and this is a good one.

I guess that there are a few ways to think about this, and I'm wondering which you have in mind.

One would be to have some examples that we consider to be known or expected and treat this as maybe a testing situation where we develop/run queries over time and watch our performance as in regression testing.

Another would be to choose something actually unknown and have more of a person/group evaluating results for soft agreement.

Both have their merits and difficulties, I think.

**Reply by andrewsu** [reply]
I personally think we need a solution in place for the first interpretation (something more like automated regression testing).  I think the second interpretation (human review) is of course valuable -- different teams have been doing this to different degrees, and consortium-wide coordination would of course be useful.  But for my interest in optimizing sorting/filtering strategies, I'm more interested in automated testing, and I don't think that's an area we've explored enough yet...  My two cents...

**Reply by atropsha** [reply]
Dear All: Chris alerted me to this discussion. The topic is dear to my heart as for the past three and a half years my group has been working on the project called Infohub for Rare Diseases (link is below). Several thoughts. 
-	We have a published a paper (https://pubmed.ncbi.nlm.nih.gov/29330123/) including a database of “repurposed” drugs (loosely defined because it is all bibliometric, not FDA approved). We called a drug “repurposed” if a “new” indication was suggested in a paper published no less than 3 years after the drug was officially designated for some disease. Of course someone mentioning a new use in a paper means almost nothing as far as real use.  We could filter that database for rare diseases if of interest. 
-	We have data in Infohub for Rare Diseases (https://rarediseases.oscar.ncsu.edu/) that enables quick navigation to what’s known (if anything) about drugs tried for rare diseases. For instance, typing the word “metabolic” in the search window brings “Disorder of asparagine metabolism” as the first (alphabetical) hit. This search finds a rare disease in this category, i.e., Congenital microcephaly-severe encephalopathy-progressive cerebral atrophy syndrome). Further look at the data under Research tab, finds that there are no approved drugs in the FDA Orange Book but there is also info that can be extracted from  FDA Orange Book saying that Insulin and Valproate have been suggested for Asparagine synthetase deficiency

So I fully agree with Chris that we should be better off focusing on one (or few) diseases, maybe  a cluster of disease with similar symptoms; perhaps work with the team behind GARD; maybe connect with a clinician(s) that understand a specific disease and see if whatever we could find makes sense in the clinic. Just thoughts. The tools I mentioned above could be used for such searches. 

Cheers, =Alex=

**Reply by southalln** [reply]
[OOPD-22-02-21.xlsx](https://github.com/NCATSTranslator/Relay/files/6025556/OOPD-22-02-21.xlsx)
This is a list of products with an orphan designation at FDA - and I've done a crude mapping to disease term (using GARD infrastructure) and then a really rough mapping from drug to target. This highlights some obvious flaws, e.g. not being able to say which gene is responsible for its pharmacological action for a given disease (when the drug targets multiple targets).

**Reply by atropsha** [reply]
Noel: thank you for sharing!  This is a large collection...much bigger than i expected. I think one of possibe initial uses is to (as we touched on above) identify/decide on a cluster of diseases and already associated drugs (if such exist) and then (i) address the gaps that you identifued such as missing genes (or COPs) and (ii) search for additional candidates for repurposing. 

Sensible?

**Reply by andrewsu** [reply]
@southalln: I see where you downloaded the original version of that file from OOPD https://www.accessdata.fda.gov/scripts/opdlisting/oopd/index.cfm. Is the code/method you used to do the mappings (to GARD, UNII, UniProt, etc.) in a shareable form? 

Also, Mike mentioned our work developing DrugMechDB here https://ncatstranslator.slack.com/archives/CSS752SB0/p1615240849031900.  If we extended that curation effort to the rare disease indications you have here, then that might help with the evaluation of results that come back from Translator and facilitate the gap analysis that @atropsha mentioned...
